### ISSUE TITLE
YOLO V4 images size for better accuracy

### ISSUE THREAD

**Issue by shubham-shahh** [AlexeyAB/darknet#6758]
Hi, i hope everyone is doing well. I want to make a numberplate detector with yoloV4. my input images are of size from 200(width)x60(Height) to 400x150. my question is whar shall be my height and width in config file? i trained it with 416x416 but didn't get satisfactory results

[net]
# Testing
#batch=1
#subdivisions=1
# Training
batch=64
subdivisions=16
width=416
height=416
channels=3
momentum=0.949
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 6000
policy=steps
steps=4800,5400
scales=.1,.1

#cutmix=1
mosaic=1

#:104x104 54:52x52 85:26x26 104:13x13 for 416

plus does darknet so any data augmentation while training? if yes what are the variations?

thanks in Advance. have a nice day.

**Comment by lsd1994** [comment]
You can try 416x160.

**Comment by shubham-shahh** [comment]
> You can try 416x160.

so, is 414x414 the network's resolution or image resolution?

**Comment by lsd1994** [comment]
image resolution

**Comment by shubham-shahh** [comment]
Are you sure? if i use 608x608 will the height and width increase or black padding will be applied?

**Comment by stephanecharette** [comment]
Isn't this the 2nd issue you've opened for the same questions?

Darknet will resize the images to match the resolution.  Aspect ratio is ignored.  No black padding will be applied.

https://www.ccoderun.ca/programming/2020-09-25_Darknet_FAQ/#square_network

> plus does the darknet do any data augmentation while training? if yes what are the variations?

Data augmentation options are described here:  https://www.ccoderun.ca/darkmark/DataAugmentation.html

**Comment by shubham-shahh** [comment]
sir, I tried -show_imgs and found that yolo indeed applies some padding.
